testing/snapshot.ts

```diff
@@ -20,6 +19,19 @@ type AssertSnapshotContext = {
 
 let _assertSnapshotContext: AssertSnapshotContext;
 
+export function serialize(actual: unknown): string;
+export function serialize<T>(actual: T): string;
+export function serialize(actual: unknown): string {
```

Comment:
Why are these exported?

Reply:
Given the proposed API changes in #2135 I thought it would be nice to export the default `serialize` function so that people can wrap it when providing custom serializers to `assertSnapshot`. However, this has nothing to do with this PR so I am happy to remove the export keyword for now.

Reply:
That'd be great, thank you

Reply:
Done 👍